### ISSUE TITLE
Postinstall completes, but it has no effect

### ISSUE THREAD

**Issue by calhouncole** [blargity/react-native-schemes-manager#33]
### Steps to reproduce the behavior
I have 3 different Schemes with the associated Build Configurations:
Production: Release, Debug
Sandbox: Sandbox Release, Sandbox Debug
Dev: Dev Release, Dev Debug

Added the following to my package.json:
```
  "scripts": {
    "postinstall": "react-native-schemes-manager all"
  },
  "xcodeSchemes": {
    "Debug": [
      "Dev Debug",
      "Sandbox Debug"
    ],
    "Release": [
      "Release"
    ],
    "projectDirectory": "ios"
  }
```

When running `npm install`, the postinstall script runs fine and produces the following:

```
Hiding schemes from node_modules xcode projects.
ios/ProjectName.xcodeproj
 - [fix-script]: ios/ProjectName.xcodeproj skipped
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native-vector-icons/RNVectorIcons.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native-vector-icons/RNVectorIcons.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native-vector-icons/RNVectorIcons.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/ActionSheetIOS/RCTActionSheet.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/ActionSheetIOS/RCTActionSheet.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/ActionSheetIOS/RCTActionSheet.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/ART/ART.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/ART/ART.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/ART/ART.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/Blob/RCTBlob.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/Blob/RCTBlob.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/Blob/RCTBlob.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/CameraRoll/RCTCameraRoll.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/CameraRoll/RCTCameraRoll.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/CameraRoll/RCTCameraRoll.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/Geolocation/RCTGeolocation.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/Geolocation/RCTGeolocation.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/Geolocation/RCTGeolocation.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/Image/RCTImage.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/Image/RCTImage.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/Image/RCTImage.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/LinkingIOS/RCTLinking.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/LinkingIOS/RCTLinking.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/LinkingIOS/RCTLinking.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/NativeAnimation/RCTAnimation.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/NativeAnimation/RCTAnimation.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/NativeAnimation/RCTAnimation.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/Network/RCTNetwork.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/Network/RCTNetwork.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/Network/RCTNetwork.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/PushNotificationIOS/RCTPushNotification.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/PushNotificationIOS/RCTPushNotification.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/PushNotificationIOS/RCTPushNotification.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/RCTTest/RCTTest.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/RCTTest/RCTTest.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/RCTTest/RCTTest.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/Sample/Sample.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/Sample/Sample.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/Sample/Sample.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/Settings/RCTSettings.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/Settings/RCTSettings.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/Settings/RCTSettings.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/Text/RCTText.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/Text/RCTText.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/Text/RCTText.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/Vibration/RCTVibration.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/Vibration/RCTVibration.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/Vibration/RCTVibration.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/Libraries/WebSocket/RCTWebSocket.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/Libraries/WebSocket/RCTWebSocket.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/Libraries/WebSocket/RCTWebSocket.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/local-cli/templates/HelloWorld/ios/HelloWorld.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/local-cli/templates/HelloWorld/ios/HelloWorld.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/local-cli/templates/HelloWorld/ios/HelloWorld.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/React/React.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/React/React.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/React/React.xcodeproj
 ✔ [fix-libraries]: Debug -> Dev Debug created in node_modules/react-native/React/ReactLegacy.xcodeproj
 ✔ [fix-libraries]: Debug -> Sandbox Debug created in node_modules/react-native/React/ReactLegacy.xcodeproj
 - [fix-libraries]: Release -> Release skipped in node_modules/react-native/React/ReactLegacy.xcodeproj
```

### Expected behavior
Running my app in Xcode with either the Sandbox or Dev Schemes (which use Sandbox Debug and Dev Debug build configurations) would be able to access react natives dev tools.

### Actual behavior
I don't have access to the dev tools with those schemes. Only when I build with "Production" scheme from Xcode does it provide me with dev tools. I can't do development with our production endpoints for obvious reasons.

Thanks for the help here!

**Comment by blargity** [comment]
@calhouncole We've introduced some fixes in v1.0.2 and v1.0.3 that'll probably help here. Can you update to the latest version and let me know how you go?

**Comment by blargity** [comment]
I believe this has likely been fixed. I'll close this issue for now, but if I'm wrong and it's still around, please don't hesitate to open a new one!
